ui/admin/tests/unit/utils/param-value-finder-test.js

```diff
@@ -0,0 +1,37 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { paramValueFinder } from 'admin/utils/param-value-finder';
+
+module('Unit | Utility | param-value-finder', function (hooks) {
+  setupTest(hooks);
+
+  let router;
+
+  hooks.beforeEach(function () {
+    router = this.owner.lookup('service:router');
+  });
+
+  test('returns empty array when null or empty routeInfo object is used', function (assert) {
+    assert.expect(2);
+    let result = paramValueFinder('scopes', null);
+
+    assert.deepEqual(result, []);
+
+    result = paramValueFinder('scopes', {});
+
+    assert.deepEqual(result, []);
+  });
+
+  test('returns correct array when routeInfo object is used', function (assert) {
+    assert.expect(2);
+    let routeInfo = router.recognize('/scopes');
+    let result = paramValueFinder('scopes', routeInfo.parent);
+
+    assert.deepEqual(result, []);
+
+    routeInfo = router.recognize('/scopes/global/auth-methods');
+    result = paramValueFinder('scopes', routeInfo.parent);
+    console.log(routeInfo);
```

Comment:
Not sure, but looks like we forget this `console.log` ?

Reply:
ahh yes! great catch, thank you!